tests/test_load_from_hub.py

```diff
@@ -0,0 +1,25 @@
+import gym
+
+from huggingface_sb3 import load_from_hub, RepoId, ModelName, EnvironmentName
+from stable_baselines3 import PPO
+from stable_baselines3.common.evaluation import evaluate_policy
+
+
+def test_load_from_hub():
```

Comment:
maybe add a unit test to check that third party package names are correctly converted by the helper? (no need to download/upload anything there)